configuration/pih/htmlforms/hiv/section-pmtct-history.xml

```diff
@@ -5,129 +5,130 @@
         <ifMode mode="VIEW" include="false">
 
             #data-collection {
-                display: inline-block;
-                width: 58%;
-                vertical-align: top;
+            display: inline-block;
```

Comment:
I suspect the formatter doesn't know how to format CSS and JavaScript found within <ifMode> tags since it's non-standard and specific to HTML Form Entry.  I've always been manually adding the tabs, which, admittedly, is a big pain.

@louidorjp  @brandones  thoughts on best practices here?

Reply:
Yeah I'm not inclined to worry too much about these. Manually maintaining the correct formatting is a pain, but is preferable to having it all unreadable.